hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerData.java

```diff
@@ -89,7 +91,9 @@
   private HddsVolume volume;
 
   private String checksum;
-  public static final Charset CHARSET_ENCODING = Charset.forName("UTF-8");
+  private Long dataScanTimestamp;
```

Comment:
Can you make this a Java `Optional`. Then instead of `null` we can check for `Optional.absent`.

Reply:
Also can you add a comment stating what the number means? Is it Unix epoch?

Reply:
Thanks for the comments.  I will address these and update the pull request in the new repo.